libs/components/ag-grid/src/lib/modules/ag-grid/ag-grid-row-delete.directive.spec.ts

```diff
@@ -15,100 +15,127 @@ describe('SkyAgGridRowDeleteDirective', () => {
     fixture.detectChanges();
   });
 
```

Comment:
This test was the largest pain point, so I broke it up into smaller tests.

The issue was that the AG Grid events get triggered asynchronously in their own promise wrapper, and returning a promise from `fixture.componentInstance.filterName` et al, which didn't resolve until the event was triggered, made these tests less brittle.

It should be possible to re-combine these tests, but it was easier to work with the smaller tests.